apartments/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from . import views
+from . import views as apartment_views
 
 urlpatterns = [
-    path('update', views.updateApartment, name='apartment-update'),
+    path('update', apartment_views.updateApartment, name='apartment-update'),
+    path('register/', apartment_views.register_apartment, name='register_apartment'),
```

Comment:
is there a reason why register has a / and update doesn't?

Reply:
Not really, I thought that this is the way your supposed to write it - I think I saw a video of someone writing this way, I can change it but it works this way too, what do you think?

Reply:
i think it should be the same in both. But I won't hold it up because of that.